### ISSUE TITLE
Add support for mounting ISO as CD-Rom

### ISSUE THREAD

**Issue by poutros** [Streetwalrus/android_usb_msd#13]

**Comment by Streetwalrus** [comment]
I've come across the need for this (mostly for booting raw Windows ISOs, because MS is retarded), so it doesn't sound like a bad idea at all. Needs a bit of a refactor tho. I'll do it - eventually.

**Comment by chainsawbike** [comment]
its easy enough to do in the command prompt as root:

easiest way:
before opening USB mountr
`echo 1 > /sys/class/android_usb/android0/f_mass_storage/cdrom`
 will set it to cdrom mode until you reboot the phone or do:
`echo 0 > /sys/class/android_usb/android0/f_mass_storage/cdrom` once you "unmount" it in the app
to set it back to mass storage mode

otherwise you just need to disable and enable the usb "device" for it to take effect ie:
`echo 0 > /sys/class/android_usb/android0/enable`
`echo 1 > /sys/class/android_usb/android0/f_mass_storage/cdrom`
`echo 1 > /sys/class/android_usb/android0/enable`

note that i am using the older kernel that only handles one export at a time, newer kernels may have things at a different location or behave slightly differently.

**Comment by Streetwalrus** [comment]
Yeah the new kernels do it slightly differently, I know how to do it but my phone has a strange bug that doesn't let me change the LUN list after setting it for the first time. I'll have to work around this (probably by simply registering two LUNs, one for regular mode, one for CDROM mode).
